react/week1/todolist/src/Todolist.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const TodoListItems = (props) => {
+  const {description, deadline} = props.todo
+  return (
+    <li>
+      <h1>
+        {description} : {deadline}
+      </h1>
+    </li>
+  );
+};
+
+const TodoList = () => {
+  return (
+    <ul>
+      {todos.map((todo) => {
+        return <TodoListItems todo={todo} />;
+      })}
+    </ul>
+  );
+};
+
+const todos = [
+  { description: "Get out of bed", deadline: "Wed Sep 13 2017" },
+  { description: "Brush teeth", deadline: "Thu Sep 14 2017" },
+  { description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+];
```

Comment:
As an alternative, you could also keep them in a separate json file, which you could then simply load by
```import todos from "./todosjson";```
It is of course fine this way, but in a more realistic scenario you wouldn't have hard-coded data in your js files

Reply:
Same here. Thank you for the feedback.